OtripleS.Web.Api/Brokers/Storage/StorageBroker.Fees.cs

```diff
@@ -0,0 +1,51 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using OtripleS.Web.Api.Models.Fees;
+
+namespace OtripleS.Web.Api.Brokers.Storage
+{
+    public partial class StorageBroker
+    {
+        public DbSet<Fee> Fees { get; set; }
+
+        public async ValueTask<Fee> InsertFeeAsync(Fee fee)
+        {
+            var feeEntityEntry = await this.Fees.AddAsync(fee);
```

Comment:
Instead of using `this.Fees` we changed our standard a bit - so all your broker methods now needs to be as follows.

```csharp
using var broker = new StorageBroker(this.configuration);
EntityEntry<Fee> feeEntityEntry = await broker.Fees.AddAsync(fee);
await broker.SaveChangesAsync();

return feeEntityEntry.Entity;

```